tutorials/3d/csg_tools.rst

```diff
@@ -153,19 +154,19 @@ Create a ``CSGCombiner`` and name it ``lamp``.
 
 A lamp consists of 3 parts: the stand, the pole, and the lampshade.
 
-Create a ``CSGCylinder`` enable option ``cone`` and make it the stand. Create another ``CSGCylinder`` and adjust the dimensions to use it as a pole.
+Create a ``CSGCylinder``\ , enable the ``cone`` option and make it the stand. Create another ``CSGCylinder`` and adjust the dimensions to use it as a pole.
 
 .. image:: img/csg_lamp_pole_stand.png
 
-We will use ``CSGPolygon`` for the lampshade. Use ``Spin`` mode for the ``CSGPolygon`` and draw a trapezium in front view, this shape will extrude around the origin and form the lampshade.
+We will use ``CSGPolygon`` for the lampshade. Use ``Spin`` mode for the ``CSGPolygon`` and draw a trapezium in ``Front View`` (numeric keypad 1); this shape will extrude around the origin and form the lampshade.
```

Comment:
"Trapezoid" is better, [wikipedia](https://en.wikipedia.org/wiki/Trapezium):
> * Outside the US and Canada: a quadrilateral with at least one pair of parallel sides (known in the US as a **trapezoid**)
> * In the US and Canada: a quadrilateral with no parallel sides (known elsewhere as a general irregular quadrilateral)

Reply:
Seems to be a North America vs Rest of the world issue... As we decided to take American English (for better or worse) as reference language for the docs, we should indeed use "trapezoid".

Additional perk, it's consistent with the French "trapézoïde" or the German "Trapezoid", so it might be easier to understand for non-native readers.

Reply:
> Additional perk, it's consistent with the French "trapézoïde" or the German "Trapezoid", so it might be easier to understand for non-native readers.

But not with the Spanish "trapezoide" which has the meaning of the North American English "trapezium" (irregular quadrilateral).

Well, f- it, let's just stick to en_US :)